### ISSUE TITLE
EYE dependencies missing and remaining Python 2.7 formatting in EYE

### ISSUE THREAD

**Issue by MichlF** [dvanmoorselaar/open_dvm#1]
Missing dependency in EYE.py -> pygazeanalyzer

In EYE.py, there's a few print statements that are still in python 2.7 style.

If I find more I can also fork it, so you can more quickly implement the fixes...

**Comment by dvanmoorselaar** [comment]
Yes. You should download the pygazeanalyzer (https://github.com/esdalmaijer/PyGazeAnalyser) and save in your python environment used for the analysis.
